policyengine_us/parameters/gov/states/il/rta/cta/monthly_pass_cost/full_fare.yaml

```
@@ -0,0 +1,11 @@
+description: The Chicago Transit Authority provides the full fare 30-Day CTA/Pace Pass at this amount. 
```

```suggestion
description: The Chicago Transit Authority provides the full fare 30-Day CTA/Pace Pass at this amount of x. 
```

Fill in the x - what do they get?

---

same in other parameters